Gemfile.lock

```diff
@@ -144,31 +144,31 @@ GEM
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     aes_key_wrap (1.1.0)
-    ahoy_matey (3.0.5)
+    ahoy_matey (3.1.0)
       activesupport (>= 5)
       device_detector
       geocoder (>= 1.4.5)
       safely_block (>= 0.2.1)
     american_date (1.1.1)
     android_key_attestation (0.3.0)
     ast (2.4.1)
-    autoprefixer-rails (10.0.3.0)
+    autoprefixer-rails (10.1.0.0)
       execjs
     awrence (1.1.1)
     aws-eventstream (1.1.0)
-    aws-partitions (1.400.0)
+    aws-partitions (1.411.0)
     aws-sdk-cloudwatchlogs (1.38.0)
       aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-core (3.109.3)
+    aws-sdk-core (3.110.0)
```

Comment:
heads up/reminder, a small AWS SDK core bump ran into proxy issues because it hit a different endpoint for some requests, so definitely make sure to test this is a deployed env

Reply:
yep, aiming to do these early in the cycle to get some time in the sandbox